### DISCUSSION TITLE
Issue with Structure Pane Not Redirecting to Selected TeX Files

### DISCUSSION THREAD

**Discussion by Fontanapink** [texstudio-org/texstudio#3480] [Q&A] [answered]
I'm experiencing an issue with TeXstudio where the Structure Pane no longer redirects the editor view to the selected TeX files as it used to. Previously, I could click on different TeX files listed in the Structure Pane, and the editor would automatically navigate to the corresponding section in the document. This functionality has suddenly stopped working.

I can click on the different _sections_  within the different opened tex files, and the editor will re-direct me to the corresponding correct subsection. That still works. However, sometimes I come back and forth between two documents, and I need to go back to exactly where I left the editor view before. I used to be able to do this just by using the Structure Pane and clicking on the name of the different files. Now I can only replicate this if I use the tabs in the editor window itself. 

Is there a known solution or setting that I might be missing to restore this functionality? Any guidance or suggestions would be greatly appreciated.

Thank you!

**Comment by octaeder** [comment]
please check if this is your issue: https://github.com/texstudio-org/texstudio/issues/3473

**Reply by Fontanapink** [reply]
It is! But that one is closed with no solution. Does that mean it has been resolved and I should wait for the next release?

**Reply by octaeder** [reply, marked as answer]
Has been closed by
![grafik](https://github.com/texstudio-org/texstudio/assets/102688820/3401ad6f-4749-4c4e-b1ae-96b72858343e)
So will be fixed with next release.

**Reply by Fontanapink** [reply]
Thank you
